client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-url/index.tsx

```diff
@@ -7,23 +7,28 @@ import { useAnalyzeUrlQuery } from 'calypso/data/site-profiler/use-analyze-url-q
 import { useSiteSlug } from 'calypso/landing/stepper/hooks/use-site-slug';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import wpcom from 'calypso/lib/wp';
-import type { Step } from '../../types';
+import type { StepProps } from '../../types';
 import type { UrlData } from 'calypso/blocks/import/types';
 
 import './style.scss';
-interface Props {
+
+interface SourceSiteInputProps {
 	hasError?: boolean;
 	onComplete: ( siteInfo: UrlData ) => void;
+	title?: string;
+	subTitle?: string;
 }
 
-export const SourceSiteInput: FC< Props > = ( { onComplete } ) => {
+export const SourceSiteInput: FC< SourceSiteInputProps > = ( { onComplete, ...options } ) => {
 	const translate = useTranslate();
 	const [ siteURL, setSiteURL ] = useState< string >( '' );
 
-	const title = translate( 'Share your site address' );
-	const subtitle = translate(
-		"Let's get your migration started. Please share your site address so we can review your site and begin your migration."
-	);
+	const title = options.title ?? translate( 'Share your site address' );
```

Comment:
One thing I noticed here different from the [other PR](https://github.com/Automattic/wp-calypso/pull/93465) is the `??` or `||`. I usually prefer the `??` as you used here to avoid the falsy cases, but I'm fine with both.

Reply:
I am going to update the other PR after merge this one.